tools/packaging/kernel/configs/fragments/x86_64/tdx/tdx.conf

```diff
@@ -10,3 +10,4 @@ CONFIG_VIRT_DRIVERS=y
 CONFIG_X86_5LEVEL=y
 CONFIG_X86_INTEL_MEMORY_PROTECTION_KEYS=y
 CONFIG_X86_PLATFORM_DEVICES=y
+CONFIG_TMPFS_XATTR=y
```

Comment:
@JakubLedworowski, please, create a new file under https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kernel/configs/fragments/common/confidential_containers and add this there.